diktat-gradle-plugin/settings.gradle.kts

```diff
@@ -0,0 +1,6 @@
+pluginManagement {
+    repositories {
+        mavenCentral()
+        gradlePluginPortal()
```

Comment:
Bintray was down, and suddenly it turned out that gradle still uses it. It won't hurt to keep this code until they remove these redirections, even though functional tests still won't work if bintray is down once again

Reply:
It worked fine, because either gradle plugin portal stores some artifacts on its own server, or bintray was up most of the time. Bintray is still available read-only, as they claim, so I think we can wait for solution on gradle's side.